### PR TITLE
src: mail: Add 'additional_emails' configuration

### PR DESCRIPTION
**Pull request by davidbtadokoro**
`kw mail` already has a configuration to automatically block emails as recipients named 'blocked_emails'. An opposite configuration, to automatically add emails as recipients doesn't exist, though.

In this context, add the configuration 'additional_emails' to `kw mail` to automatically add pre-determined recipients.

### CODE REVIEW

File: etc/mail.config
Comment (rodrigosiqueira) [inline etc/mail.config:12]: I don't think this field expresses the idea behind it really well. Maybe we can try something like `demails_always_included` (it is still a bad name, it is just an example)...

We also should have one for `to` and another one for `cc`.

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]
In general, it lgtm. Remember to add a test and reference @melissawen issue in the commit message. Also, expand the man documentation.

**Comment by davidbtadokoro** [comment]
Hey @rodrigosiqueira, thanks for the feedback!
I updated the PR addressing your comments.
In respect to the configurations name, I used `default_to_recipients` and `default_cc_recipients`. What do you think?

**Comment by melissawen** [comment]
names sounds good to me.

**Comment by rodrigosiqueira** [comment]
Overall, it lgtm. @melissawen I'll wait for your okay for this change before I merge it.

**Comment by melissawen** [comment]
Just tested and it works as expected. I noticed that when I add the same e-mail address to `default_to_recipients` and `default_cc_recipients` it doesn't repeat and the list of e-mails in `To:` is prioritized. All good, you can add my t-b:
melissa.srw \<at\> gmail.com

**Pull request closed without merging**
